universal-application-tool-0.0.1/app/forms/MultiOptionQuestionForm.java

```diff
@@ -143,27 +143,40 @@ public QuestionDefinitionBuilder getBuilder() {
       predicateBuilder.setMaxChoicesAllowed(getMaxChoicesAllowed());
     }
 
-    ImmutableList.Builder<QuestionOption> questionOptions = ImmutableList.builder();
+    ImmutableList.Builder<QuestionOption> questionOptionsBuilder = ImmutableList.builder();
     Preconditions.checkState(
         this.optionIds.size() == this.options.size(),
         "Option ids and options are not the same size.");
 
+    if (this.optionIds.size() > 0) {
+      setNextAvailableId(this.optionIds.get(0) + 1);
```

Comment:
May be it will be easier to use :  setNextAvailableId(this.optionIds.size()) because the Ids start from 0.

Reply:
I'm not sure setting the next available as the size will always work because IDs aren't guaranteed to start at 0. I actually am thinking based on these comments that maybe I should just delete this line. It is starting to feel completely unncessary

Reply:
Okay, went ahead and removed those lines. Let me know what y'all think

Reply:
False alarm, I actually brought those lines back. I realized that if there are any IDs set I want that to be the next available ID before going into the for loop, so that I'm not constantly comparing againt 0L. Let me know if that makes sense. I can comment in the code also if that would help.